src/components/itemHelper.js

```diff
@@ -331,6 +331,17 @@ export function supportsMediaSourceSelection (item) {
     return true;
 }
 
+export function sortTracks (trackA, trackB) {
+    let cmp = trackA.IsExternal - trackB.IsExternal;
+    if (cmp != 0) return cmp;
+    cmp = trackB.IsForced - trackA.IsForced;
+    if (cmp != 0) return cmp;
+    cmp = trackB.IsDefault - trackA.IsDefault;
+    if (cmp != 0) return cmp;
+
+    return 0;
```

Comment:
I shuffled the tracks to test the PR. We could use `Index` as the last ordering field, if this doesn't hurt your goal.
```suggestion
    return trackA.Index - trackB.Index;
```

Reply:
I don't realy care, did it provide a better result in some way in your test?

Reply:
After shuffling, I get tracks unordered by `Index`. In my case, this restores the original order.

At the same time, the server response is already ordered by `Index` (after your investigation). So maybe this is unnecessary or just to be sure.